debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -500,8 +500,7 @@ protected boolean isGeometryCollection(String upperCaseTypeName) {
     }
 
     protected List<String> extractEnumAndSetOptions(Column column) {
-//        return MySqlDdlParser.parseSetAndEnumOptions(column.typeExpression());
-        return MySqlAntlrDdlParser.parseSetAndEnumOptions(column.typeExpression());
```

Comment:
@Naros, sorry, one more thing: it seems the method itself could be removed now as its unused?

Reply:
I left it there because {{MySqlDdlParser#assertParseEnumAndSetOptions}} still referenced it.  In hindsight, do you think it makes more sense to consolidate the enum/set test methods into one of the classes, marking the enum/set test methods as skipped by the legacy-parser and only test them when we use the antlr parser?

Reply:
`MySqlDdlParserTest` invokes `MySqlDdlParser#parseSetAndEnumOptions()`, the same method on `MySqlAntlrDdlParser` is unused now.

I've removed it and pushed to master. I don't think more consolidation is needed, as of 0.10 we'll remove the entire legacy parser structure.